### ISSUE TITLE
Add how it works visuals to `Private Key Management` sections

### ISSUE THREAD

**Issue by Bosch-0** [BitcoinDesign/Guide#561]
Each section in the private key management chapter has a 'How it works' section. Currently these sections are all just text based but as designers I think we should be showing rather than telling. I propose we include a Bitcoin application carousel using the UI kit showing how the PKM scheme works to each `How it works` section in the following PKM pages:

- [ ] Automatic cloud backup
- [ ] Manual backup
- [ ] External signers
- [ ] Multi-Key

**Comment by GBKS** [comment]
What do you think of using diagrams instead of screen mockups? When I tried to figure out what user interactions multi-key wallets require, I created diagrams like the one below, which really helped me understand things. Then we can keep UI mock-ups for these schemes in the case studies in the context of specific products and use cases.

![multisig-wallet-setup](https://user-images.githubusercontent.com/695901/141737744-04db4a54-d518-4f3d-aef5-f768e7bf64e6.jpg)

**Comment by Bosch-0** [comment]
Yep diagrams are also great, fits in well with the PKM overview page which uses these visuals also.

**Comment by pavlenex** [comment]
Concept ACK, after reviewing these pages it really felt like we should provide graphics in the how it works section. @Bosch-0 are you taking on this one?

**Comment by Bosch-0** [comment]
Yep I can take on this one! Christoph and Daniel already has a lot of the foundations laid out so shouldn't take long

**Comment by GBKS** [comment]
This is now done for the multi-key page via #1064. I will take a look at the other pages to see if diagrams make sense. Those flows are a lot simpler, so it may not be warranted. But I'll take a look.
